.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -27,16 +27,16 @@ body:
     attributes:
       label: Which package are you using?
       options:
-        - @sentry/angular
```

Comment:
Why this change?

Reply:
It can't compile yml when there is `@` in front of it

Reply:
Then let’s add quotes to force it to become a string instead of making yaml coerce it?

Reply:
That actually works. Just tested it.